client/tools/mgr-push/mgr-push.spec

```diff
@@ -21,13 +21,16 @@
 %define oldname rhnpush
 %define oldversion 5.5.114
 %{!?pylint_check: %global pylint_check 0}
+%global __python to /usr/bin/python2
```

Comment:
Is the `to` correct here?

Reply:
Fixed. As this only impacts only RHEL < 8 builds, this was not detected.